icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -6119,25 +6169,25 @@ void NumberFormatterApiTest::testIssue22378() {
         {"fr_FR@mu=badvalue", u"23\u202F\u00B0C"},
     };
 
-    UnlocalizedNumberFormatter formatter = NumberFormatter::with()
-            .usage("weather")
-            .unit(MeasureUnit::getCelsius());
+    UnlocalizedNumberFormatter formatter =
+        NumberFormatter::with().usage("weather").unit(MeasureUnit::getCelsius());
     double value = 23.0;
 
     for (const auto &testCase : testCases) {
         std::string localeId = testCase.localeId;
         const Locale locale = (localeId.find("@") != std::string::npos)
-                ? Locale(localeId.c_str())
-                : Locale::forLanguageTag(localeId, status);
-        UnicodeString actualFormat = formatter.locale(locale)
-                .formatDouble(value, status)
-                .toString(status);
+                                  ? Locale(localeId.c_str())
+                                  : Locale::forLanguageTag(localeId, status);
+        UnicodeString actualFormat =
+            formatter.locale(locale).formatDouble(value, status).toString(status);
         assertEquals(u"-u-mu- honored (" + UnicodeString(localeId.c_str()) + u")",
-                testCase.expectedFormat, actualFormat);
+                     testCase.expectedFormat, actualFormat);
     }
 
-    UnicodeString result = formatter.locale("en-US").formatDouble(value, status).getOutputUnit(status).getIdentifier();
-    assertEquals("Testing default -u-mu- for en-US", MeasureUnit::getFahrenheit().getIdentifier(), result);
+    UnicodeString result =
+        formatter.locale("en-US").formatDouble(value, status).getOutputUnit(status).getIdentifier();
+    assertEquals("Testing default -u-mu- for en-US", MeasureUnit::getFahrenheit().getIdentifier(),
+                 result);
```

Comment:
This is one of a number of cases where the code seems to have been reformatted with no substantive change.  Those kinds of things make the diff a lot harder to read.  Is the reformatting necessary?

Reply:
I agree with you it is really annoying when the IDE format some part that are not necessary.

Sorry that I did not revert them, I am reverting them now.

Reply:
Thank you!